.github/CODEOWNERS

```
@@ -0,0 +1,294 @@
+/openstack/bastion                                     @fwiesel
+/openstack/billing                                     @majewsky @SuperSandro2000 @VoigtS @Nuckal777
+/openstack/blackbox                                    @artherd42 @fwiesel @m-kratochvil
+/openstack/castellum                                   @majewsky @SuperSandro2000 @VoigtS @Nuckal777
+/openstack/cbshelf                                     @corey-aloia
+/openstack/cc3test                                     @artherd42 @thgrs @ashifnihalb @Carthaca
+/openstack/cc3test-seeds                               @artherd42 @thgrs @fwiesel
+/openstack/cerebro                                     @viennaa
+/openstack/cinder                                      @joker-at-work @hemna @fwiesel @dusandordevicsap
+/openstack/content-repo                                @majewsky @SuperSandro2000 @VoigtS @Nuckal777
+/openstack/cronus                                      @kayrus @dhalimi1975 @corey-aloia @fwiesel
+/openstack/cronus-seed                                 @kayrus @dhalimi1975
+/openstack/db-backup-v2                                @databus23 @fwiesel
+/openstack/designate                                   @galkindmitrii @fwiesel @mutax @kpawar-sap @vssldmtrv @mbrowny @SoenkeL
+/openstack/designate-nanny                             @mutax @vssldmtrv @mbrowny @SoenkeL @galkindmitrii
+/openstack/domain-seeds                                @BerndKue @Carthaca @fwiesel @Kuckkuck @rajivmucheli
+/openstack/elektra                                     @andypf @databus23 @ArtieReus @hgw77
+/openstack/glance                                      @fwiesel @galkindmitrii @rajivmucheli @Carthaca
+/openstack/grafanasix                                  @thgrs @richardtief
+/openstack/hermes                                      @notque @Kuckkuck @IvoGoman @fwiesel
+/openstack/infra-seeds/kubernetes                      @BugRoger  # old
+/openstack/ironic                                      @stefanhipfel @BerndKue @fwiesel @sandzwerg
+/openstack/keppel                                      @majewsky @SuperSandro2000 @VoigtS @Nuckal777
+/openstack/keppel-trivy                                @majewsky @SuperSandro2000 @VoigtS @Nuckal777
+/openstack/keystone                                    @galkindmitrii @Carthaca @bbobrov @fwiesel @rajivmucheli
+/openstack/kos-operator                                @stefanhipfel @databus23
+/openstack/kubernetes                                  @fwiesel
+/openstack/kubernikus                                  @fwiesel @jknipper
+/openstack/limes                                       @majewsky @SuperSandro2000 @VoigtS @Nuckal777
+/openstack/lyra                                        @databus23 @ArtieReus
+/openstack/maia                                        @notque @artherd42 @viennaa @richardtief @Kuckkuck @dhalimi1975
+/openstack/manila*                                     @Carthaca @chuan137 @galkindmitrii @kpawar-sap
```

```suggestion
/openstack/manila*                                     @Carthaca @chuan137 @galkindmitrii @kpawar-sap @sumitarora2786 @crenduchinta88
```

Adds Sumit and Chandra from Storage Team to Manila CodeOwners